dbt/include/spark/macros/adapters.sql

```diff
@@ -387,6 +387,7 @@
         "identifier": tmp_identifier
     }) -%}
 
+    {%- set tmp_relation = tmp_relation.include(database=false, schema=false) -%}
```

Comment:
For posterity, I believe this change was made to avoid including the database and schema when rendering the create statement for a view, which was necessary by the unit testing framework.

I believe it could be implemented more precisely by the changes here: https://github.com/dbt-labs/dbt-spark/pull/978/files#diff-786bb6587e86e50a2d01888eb2d4a5257e9a0025f75379214fa93cd5a033c9fbR141